### PR TITLE
Release v0.0.1 for LangChain-Redis Partner Package

### PR DESCRIPTION
**Pull request by bsbodden**
What's inside:
- Fix for correctly handle Index params when building from a RedisVL IndexSchema
- Reporting of library name and version along with RedisVL and redis-py versions
- General code clean up

### CODE REVIEW

File: libs/redis/langchain_redis/config.py
Comment (tylerhutcherson) [inline libs/redis/langchain_redis/config.py:102]: I think we're holding off on JSON support for the time being correct? @bsbodden

**Review by tylerhutcherson — APPROVED** [review]
Once we remove the json storage type reference, I think we're good from my end.

**Review by efriis — COMMENTED** [review]
woohoo! lgtm from my end, and will defer to tyler/redis side comments :)

**Pull request merged**
